### ISSUE TITLE
[bug] Friendica no user avatar and account name

### ISSUE THREAD

**Issue by SadmL** [superseriousbusiness/gotosocial#3355]
### Describe the bug with a clear and concise description of what the bug is.

Friendica accounts shows only `@username` and blank (one of defaults, changing sometimes) avatar.

### What's your GoToSocial Version?

0.17.0-SNAPSHOT 713e733

### GoToSocial Arch

amd64 Binary

### What happened?

X

### What you expected to happen?

See account avatar and name

### How to reproduce it?

Try see https://fabler.su/profile/ludivokrug for example, my result is here https://github.com/friendica/friendica/issues/14316#issuecomment-2306590828

### Anything else we need to know?

About avatar, https://github.com/friendica/friendica/issues/14316 may be related

**Comment by NyaaaWhatsUpDoc** [comment]
this is definitely a friendica issue i'm afraid. i get the following results:

```
curl -H 'Accept: application/activity+json' 'https://fabler.su/profile/ludivokrug' | jq .image
null

curl -H 'Accept: application/activity+json' 'https://fabler.su/profile/ludivokrug' | jq .icon
null
```

**Comment by NyaaaWhatsUpDoc** [comment]
the above linked issue also confirms this returned data. friendica isn't providing any avatar / header images for gotosocial to dereference.

**Comment by jyrgi66** [comment]
Is this really a friendica issue? I see this behaviour on all friendica accounts and it seems only GTS hass issues with this. This issue is not visible with other fediverse SW like Mastodon, Iceshrimp, Misskey.

**Comment by SadmL** [comment]
This profile fetches fine, so...
![Image](https://github.com/user-attachments/assets/d5a42750-9e3b-41ee-a398-6ac7ffb5c0a9)

But sometimes his avatar and account name disappears.

**Comment by jyrgi66** [comment]
Yes I can confirm this profile being showing up correctly. Sorry for the confusion.
